### ISSUE TITLE
Documentation: Request object API documentation refers to "trusted_hosts" - not clear

### ISSUE THREAD

**Issue by dannystaple** [pallets/flask#2987]
### Expected Behavior

At the flask documentation for the request context/object http://flask.pocoo.org/docs/1.0/api/#incoming-request-data many items in the section say "see also **trusted_hosts**". For example http://flask.pocoo.org/docs/1.0/api/#flask.Request.host.

This should be a link, or obvious where to follow to understand what this is.

### Actual Behavior

There isn't a link, and it's not clear what the see also is referring to (perhaps in werkzeug?)

### Environment

http://flask.pocoo.org/docs/1.0/api/#incoming-request-data

**Comment by lepture** [comment]
weird. It is link in werkzeug: http://werkzeug.pocoo.org/docs/0.14/wrappers/#werkzeug.wrappers.BaseRequest.host

**Comment by davidism** [comment]
Related to #1852. I don't think there's much we can do about this, unless there's something we can adjust with Sphinx's intersphinx extension.

**Comment by dannystaple** [comment]
Could it just be a plain old hyperlink in rst ?

**Comment by davidism** [comment]
No, because we would have to copy all the docs from Werkzeug to insert that.
